### PR TITLE
Added events actions for registration, tickets etc

### PR DESCRIPTION
**Pull request by ab-sahira**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Almabase/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,4779 +1,7275 @@
...
+                      "registrant_id": {
+                          "type": "string",
+                          "x-ms-summary": "Zoom Registrant ID"
+                      },
+                      "response": {
+                          "type": "object",
+                          "x-ms-summary": "Response from zoom"
+                      }
+                  },
+                  "type": "obj
```
Comment (v-EgorKozhadei) [inline certified-connectors/Almabase/apiDefinition.swagger.json:6511]: @ab-sahira make sure that everything is grammatically correct and is free of typographical errors. You refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Comment (v-EgorKozhadei) [inline certified-connectors/Almabase/apiDefinition.swagger.json:6581]: @ab-sahira please make sure that everything is grammatically correct and is free of typographical errors. You refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @ab-sahira! Please look through the comments, thank you!

**Comment by Amjed-Ayoub** [comment]
Hello @ab-sahira,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by v-EgorKozhadei** [comment]
Hello @ab-sahira, 

I hope you are doing well. 

I am following up to see if you need any help with updating your connector artifacts. 

Please do not hesitate to reach out to us if you have any questions as we are always here to help.

Looking forward to hearing from you. 

Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hello @ab-sahira,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by sriyen-msft** [comment]
@ab-sahira We are closing your PR since there is no response from your side. You can reopen the same PR whenever you are ready for certification of your connector, thanks.

**Pull request closed without merging**